src/ZIP/src/fmi_miniunz.c

```diff
@@ -31,7 +31,7 @@
         #endif
 #endif
 
-#if defined(__APPLE__) || defined(__HAIKU__) || defined(MINIZIP_FOPEN_NO_64)
+#if defined(__APPLE__) || defined(__HAIKU__) || defined(MINIZIP_FOPEN_NO_64) || defined(_WIN32)
```

Comment:
Are there issues with win32 this tries to solve? Note that _WIN32 is set also for win64, and likely we still want to use fopen64 etc for win64.

https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170&redirectedfrom=MSDN

Reply:
maybe I missed something